### ISSUE TITLE
Switch from iCloud login to email login

### ISSUE THREAD

**Issue by xplosionmind** [omnivore-app/omnivore#1804]
I signed up to Omnivore back when the only possibility to subscribe was iCloud. Now that email login is supported, I would like to unlink my iCloud account and use Omnivore as a standalone service. It would also be enough, as a first step, if I could add an email to my current account, so that I do not need to login into iCloud everytime.

**Comment by jacksonh** [comment]
Yes, now that we have some infrastructure for deploying tools we should try and do this.

Migrating from social to email based login seems possible, having both would probably cause some issues.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
